pyproject.toml

```
@@ -75,6 +75,7 @@ dependencies = [
     "pulsar-galaxy-lib>=0.15.10",
     "pycryptodome",
     "pydantic[email]>=2.7.4",  # https://github.com/pydantic/pydantic/pull/9639
+    "pydicom",
```

```suggestion
    "pydicom; python_version >= '3.10'",
```

Would that work for you?